.travis.yml

```diff
@@ -0,0 +1,18 @@
+
+language: node_js
+node_js:
+  - "lts/*"
+dist: trusty
+sudo: false
+cache:
+  directories:
+  - node_modules
```

Comment:
Does travis CI allow you to regex this so it caches all node_modules? Easier to maintain as we add packages.

Reply:
I can look into this and add a feature task for it later.  This is how create-React-app was doing it.  But, I like your idea.

Reply:
+1; up to you whether you want to add the task or do it now. I'd pri the task accordingly though so we can clean this up quickly.

Reply:
Didn't find anything about regex.  further reading makes this sound like a bad idea with not much gain on perf if over few hundred megs.  will remove cache for now.